soc/intel/intel_adsp/ace/include/ace30_ptl/adsp_shim.h

```diff
@@ -106,6 +106,9 @@ struct ace_dfpmccu {
 
 #define ACE_CLKCTL_WOVCRO    BIT(4)	  /* Request WOVCRO clock */
 
+#define ACE_CRODIV_CARCDS_MASK		GENMASK(7, 0)
+#define ACE_CRODIV_CARCDS(x)		((x) & ACE_CRODIV_CARCDS_MASK)
+
```

Comment:
I don't get this compliance fail?
Is it required to use random indentation in headers?